pkgs/applications/office/libreoffice/online/kit-chroot-symlinks.patch

```diff
@@ -0,0 +1,81 @@
+commit a745cbdee2f4571917b420553f8cbcd5f51a4cf0
+Author: Martin Milata <martin@martinmilata.cz>
+Date:   Thu Jan 30 17:44:31 2020 +0100
+
+    tdf#129895: handle symlinks when populating chroot
```

Comment:
Did you also send this to upstream?

Reply:
yup https://gerrit.libreoffice.org/c/online/+/87749